pyproject.toml

```diff
@@ -33,6 +33,11 @@ ollama = "^0.2.0"
 openai = "^1.29.0"
 uvloop = "^0.19.0"
 httptools = "^0.6.1"
+anthropic = "^0.26.1"
+google-generativeai = "^0.5.4"
+mistralai = "^0.2.0"
+pytest = "^8.2.1"
```

Comment:
pytest is only needed for the tests, and it's already included in the "test" group, would you mind removing it from here please? :)